base_local_planner/src/odometry_helper_ros.cpp

```diff
@@ -77,6 +96,65 @@ void OdometryHelperRos::getRobotVel(tf::Stamped<tf::Pose>& robot_vel) {
   robot_vel.stamp_ = ros::Time();
 }
 
+void OdometryHelperRos::getEstimatedRobotVel(tf::Stamped<tf::Pose>& robot_vel) {
```

Comment:
Probably want to add these to the configurable parameters and also grab the accelleration limits, etc. from the local planner params.  Another option (since it looks like sort of what we are doing) is to